src/lib/redux/siteDataSlice.ts

```diff
@@ -69,9 +74,14 @@ const siteDataSlice = createSlice({
         ]
         }
       }
-      }
+    },
+    editCustomer(state, action) {
+      const index = state.currentSite.customers.findIndex((e: CustomerRecord) => e.id === action.payload.id);
```

Comment:
Can you put a note that this doesn't currently work for customers created while offline since they don't have an ID? We can address it in a future PR but we should definitely put that down so we remember.